internal/server/sshd.go

```diff
@@ -231,6 +230,20 @@ func StartSSHServer(sshListener net.Listener, privateKey ssh.Signer, insecure bo
 			}, nil
 
 		},
+		PasswordCallback: func(conn ssh.ConnMetadata, password []byte) (*ssh.Permissions, error) {
```

Comment:
We're not adding password based authentication to the RSSH server.

Reply:
#79 but i think you already agree on this..

Reply:
No. I was agreeing that requiring a password before a user can **connect** to a device is a good idea.